charts/kubearchive/templates/_helpers.tpl

```diff
@@ -3,17 +3,19 @@ Copyright KubeArchive Authors
 SPDX-License-Identifier: Apache-2.0
 */}}
 
-{{/*
-Create environment variables for OpenTelemetry if .observability is set to true. Otherwise set KUBEARCHIVE_OTEL_ENABLED=false.
-This tells the OpenTelemtry instrumentation if it should start or not.
-*/}}
+
+{{/* Create environment variables for OpenTelemetry */}}
 {{- define "kubearchive.v1.otel.env" -}}
-{{- if .observability -}}
-- name: KUBEARCHIVE_OTEL_ENABLED
-  value: "true"
-{{- else -}}
-- name: KUBEARCHIVE_OTEL_ENABLED
-  value: "false"
-{{- end -}}
+{{- $enabled := .Values.observability.enabled -}}
+{{- $endpoint := .Values.observability.endpoint -}}
+
+{{/* If 'enabled' we deploy a traces collector, so override the endpoint to use it */}}
+{{- if and (eq $enabled true) -}}
```

Comment:
Why is this "if and"? Shouldn't it just be `if (eq $enabled true)`?

Reply:
Good catch! You are right. It was a leftover from the changes, I removed the `and`.